### ISSUE TITLE
 Feedback to 2_instance_segmentation.ipynb

### ISSUE THREAD

**Issue by Steffen-Wolf** [dlmbl/instance_segmentation#1]
People were confused with the BCEWithLogitsLoss. In the previous exercise (05) they were introduced to  the BCELoss and have never heard of logits. In this exercise, they are now tasked with defining the loss function and BCE loss seems to be the right choice. The confusion is created because  the training is predefined and requires the use of BCEWithLogitsLoss  without giving a hint. Since both loss functions have the same signatures the code can be executed and gives a cryptic cuda error message. I suggest we either give a hint, introduce logits here or make the taks a multiple choice with only one BCE loss.

**Comment by pattonw** [comment]
Also no longer seems relevant. I think we just stick to BCELoss when necessary since the numerical precision advantages of using logits probably isn't necessary detail for this exercise
